site/content/api/Tenant-API.md

```diff
@@ -222,42 +238,41 @@ The table below provides an overview of the standard members defined for the JSO
+This allows for future *well-known* additions and also allows *clients* to add further information which might be e.g. relevant just for certain adapters only.
 
 ### Adapter Configuration Format
 
-When providing / updating a tenant for each *Hono protocol adapter* a configuration must be provided.
-Protocol Adapters without any configuration will be assumed as disabled for that tenant.
-
-
 The table below contains the properties which are used to configure a *Hono protocol adapter*:
 
 | Name                               | Mandatory | Type       | Default Value | Description |
 | :--------------------------------- | :-------: | :--------- | :------------ | :---------- |
-| *adapter-id*                       | *yes*     | *string*   | `null`        | The identification of the adapter which this configuration belongs to |
-| *enabled*                          | *no*      | *string*   | *true*        | If set to false the tenant is not allowed to receive / send data utilizing the given adapter. |
-| *deviceAuthentificationEnabled*   | *no*      | *string*   | *true*        | If set to false devices utilizing the given adapter are not required to authenticate themselves before sending / receiving data through the adapter. |
+| *type*                       | *yes*     | *string*   | `null`        | The type of the adapter which this configuration belongs to |
+| *enabled*                          | *no*      | *string*   | `true`        | If set to false the tenant is not allowed to receive / send data utilizing the given adapter. |
+| *device-authentication-required*   | *no*      | *string*   | `true`        | If set to false devices utilizing the given adapter are not required to authenticate themselves before sending / receiving data through the adapter. |
+
+The adapter configuration of a tenant should be evaluated by the Hono Protocol Adapters.
+If for a certain protocol adapter no configuration is present it has the same semantic as a disabled protocol adapter with default values.
 
 Additionally to the specified properties the JSON object MAY contain an arbitrary number of members with arbitrary names which can be of a scalar or a complex type. 
 
 ### Examples
 
-Below is an example for a payload defining a tenant with tenant-id `DEFAULT_TENANT` which is enabled. 
-Furthermore the tenant is allowed to use an adapter identified with the id `rest-adapter` for which devices need to use authentification. As there are not other adapters configured in the configuration, the tenant would not be allowed to use any other adapter.
+Below is an example for a payload defining a tenant with id `DEFAULT_TENANT` which is enabled. 
+Furthermore the tenant is allowed to use an adapter of thype `rest-adapter` for which devices need to use authentification. As there are not other adapters configured in the configuration, the tenant would not be allowed to use any other adapter.
```

Comment:
how about this:
Below is an example for a payload defining an *enabled* tenant with id `DEFAULT_TENANT`. 
Devices belonging to the tenant can connect to Hono via the `rest-adapter` only and are required to authenticate with the adapter on connection.

Reply:
+1 ... Will adapt it like this